openapi.yaml

```diff
@@ -12023,6 +12528,15 @@ components:
           - application/vnd.whispir.contact-v1+json
           - application/vnd.whispir.contact-v1+xml
       description: Application specific mime-type.
+    Key-Content-Type:
```

Comment:
Per the previous comment on vendor-specific MIME type deprecation, we can remove this schema.

Reply:
updated same as above